ultravox/data/datasets.py

```diff
@@ -8,70 +8,117 @@
 import os
 import tempfile
 import warnings
-from enum import Enum
-from typing import Any, Callable, Dict, List, Optional, Sequence
+from typing import Any, Dict, List, Optional, Sequence
 
-import datasets
+import datasets as hf_datasets
 import jinja2
 import librosa
 import numpy as np
-import requests
 import soundfile as sf
 import streaming as mds
 import torch
 import torch.nn.functional as F
 import transformers
+from simple_parsing import helpers
 from torch.utils import data
 
-from ultravox.data import dataset_config
 from ultravox.data import text_proc
 
 SAMPLE_RATE = 16000
 
-TRANSCRIBE_INPUT_TASK = "transcribe_input"
-TRANSCRIBE_OUTPUT_TASK = "transcribe_output"
-ANSWER_TASK = "answer"
-
-TRANSCRIBE_PROMPTS = [
-    # from Gazelle
-    "Transcribe\n<|audio|>",
-    "Transcribe exactly what is said here\n<|audio|>",
-    "Repeat exactly what is written here: <|audio|>",
-    "Write exactly what was said: <|audio|>",
-    "First listen to the clip. Then, transcribe exactly what is said. <|audio|>",
-    # from https://arxiv.org/pdf/2402.08846
-    "Transcribe speech to text: <|audio|>",
-    # from GPT-4
-    "Capture every word from the audio verbatim\n<|audio|>",
-    "Convert speech to text from audio\n<|audio|>",
-    "Listen and transcribe the complete text from audio\n<|audio|>",
-    "Record in writing what is spoken in audio\n<|audio|>",
-    "Transcribe the spoken words from audio with exact wording and punctuation\n<|audio|>",
-]
-ANSWER_PROMPTS = [
-    # from Gazelle
-    "Listen to <|audio|> and respond to it",
-    "Listen and respond: <|audio|>",
-    "Respond to <|audio|>",
-    "Respond to the user <|audio|>",
-    "<|audio|>",
-    "<|audio|>",  # repeated to emphasize not needing a prompt for Q&A tasks
-    "Respond to this question: \n<|audio|>",
-    "Continue the conversation after <|audio|>",
-    "First listen to the clip: <|audio|>\n How would you respond?",
-    "<|audio|> - respond",
-    "<|audio|>\n Respond to the question",
-]
-
 # TODO(juberti): set these in the environment so they don't need to be hard-coded here.
 os.environ["GOOGLE_APPLICATION_CREDENTIALS"] = "service_account.json"
 os.environ["GOOGLE_CLOUD_PROJECT"] = "fixie-training"
 
-
 # Silence the spurious warnings coming from the MosaicML streaming library.
 logging.getLogger("streaming.base.dataset").setLevel(logging.ERROR)
 
 
+class DatasetSplit(str, enum.Enum):
+    TRAIN = "train"
+    VALIDATION = "validation"
+
```

Comment:
I assume that we could also include a `evaluation` split, for example, for Speech Translation evaluation?

Reply:
yep, we can definitely do that. Would that typically be the "test" split?